### PR TITLE
Scheduler functions with all AWS lambda languages

### PR DESCRIPTION
**Pull request by dmkr**
This should allow scheduler to work with any language lambda has or will implement.

### CODE REVIEW

**Comment by ajmath** [comment]
Hey @dmkr, thanks for the PR and apologies for not getting back to you sooner.  Have you been using this in practice?  I'm curious if you've encountered any issues that need a follow up.

File: lib/scheduler.js [hunk 1]
```diff
@@ -54,23 +55,22 @@ class Scheduler {
...
       }
     }
     return BbPromise.resolve();
   }
 
+  _executeFunction(fName) {
+    return execSync(`serverless invoke local --function ${fName} --data {"ping":"ping"}`, {cwd: './', stdio: 'inherit' });
+  }
+  
   _requireFunction(fName) {
```
Comment (ajmath) [inline lib/scheduler.js]: It seems as if `_requireFunction` is no longer being used.  Can you remove it?  Also, please update any tests that are using this.

Comment (dmkr) [reply lib/scheduler.js]: I do need to do a clean up. Can close this pr and add a bit more functionality and the testingn testing you requested before reopening

**Comment by dmkr** [comment]
Hey @ajmath. We haven't run into any noticeable issues running it as of yet. We do turn it on and off quite a bit as it does slow down response times to run.

**Comment by dmkr** [comment]
updated with test

**Pull request merged**